### PR TITLE
@/civicsignalblog SEO

### PR DESCRIPTION
**Pull request by kilemensi**
## Description

This PR improves our implementation of SEO (and CMS) for the civicsignalblog app:

1. Moves GA analytics config to the CMS instead of ENV vars,
2. Separates SEO for normal pages (website) from posts (articles) including Article JSON-LD implementation
3. Fixes favicons and includes robots.txt (generated by our very own RoboShield),
4. Various minor UI improvements/fixes.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Screenshots

S

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: apps/civicsignalblog/src/components/Author/Author.snap.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ exports[`<Author /> renders unchanged 1`] = `
       Article by
     </div>
     <div
-      class="MuiStack-root css-3oepjh-MuiStack-root"
```
Comment (DavidTheProgrammer) [inline apps/civicsignalblog/src/components/Author/Author.snap.js:14]: Is there a way to only have these generate at build time? I think having them in source adds to the number of changes giving false positives.

**Review by DavidTheProgrammer — APPROVED** [review]
Big PR, but looks good. Maybe we should've handled the Roboshield changes in their own PR to keep the changes to a minimum but it looks fine to me. Can't see any obvious issues.

**Review by kelvinkipruto — APPROVED** [review]
LGTM

**Review by kelvinkipruto — APPROVED** [review]

File: apps/civicsignalblog/src/components/Author/Author.snap.js (hunk 1 above)
Comment (kilemensi) [reply apps/civicsignalblog/src/components/Author/Author.snap.js:14]: https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest

**Comment by kilemensi** [comment]
> Maybe we should've handled the Roboshield changes in their own PR to keep the changes to a minimum but it looks fine to me. Can't see any obvious issues.
>

RoboShield changes should have been in #841 but we missed them then...

**Pull request merged**